pubspec.yaml

```diff
@@ -1,97 +1,123 @@
 name: revanced_manager
-description: "Application to use ReVanced on Android."
-homepage: https://revanced.app
-
-publish_to: 'none'
-
+description: A Flutter application for managing ReVanced patches on Android devices.
 version: 1.24.0+101800055
+homepage: https://revanced.app
+publish_to: none
+repository: https://github.com/revanced/revanced-manager
+issue_tracker: https://github.com/revanced/revanced-manager/issues
 
 environment:
-  sdk: '>=3.7.0'
+  sdk: '>=3.7.0 <4.0.0'
   flutter: '>=3.29.0'
 
 dependencies:
-  animations: ^2.0.11
-  collection: ^1.19.0
-  connectivity_plus: ^6.1.3
-  device_apps:
-    git:
-      url: https://github.com/ponces/flutter_plugin_device_apps
-      ref: 0609662324b9e00931fccfa87e9e1b00b7610907
-  device_info_plus: ^11.3.3
-  dio: ^5.4.3+1
-  dio_cache_interceptor: ^3.5.0
-  dynamic_color: ^1.7.0
-  dynamic_themes: ^1.1.0
-  expandable: ^5.0.1
-  file_picker: ^9.0.2
   flutter:
     sdk: flutter
-  flutter_background: ^1.3.0+1
-  flutter_cache_manager: ^3.4.1
-  flutter_file_dialog: ^3.0.2
-  flutter_local_notifications: ^18.0.1
   flutter_localizations:
     sdk: flutter
+
+  # UI and Animations
+  animations: ^2.0.11
+  dynamic_color: ^1.7.0
+  dynamic_themes: ^1.1.0
+  expandable: ^5.0.1
   flutter_markdown: ^0.7.6+2
-  fluttertoast: ^8.2.12
   font_awesome_flutter: ^10.8.0
   google_fonts: ^6.2.1
-  injectable: ^2.4.0
-  intl: ^0.19.0
-  json_annotation: ^4.9.0
-  language_code: ^0.5.5
-  logcat:
+  skeletons:
     git:
-      url: https://github.com/BenjaminHalko/logcat
-      ref: 4a6d5e0e22292c8eb160cfb9365b9ea29735fd43 # Branch: master
-  package_info_plus: ^8.3.0
+      url: https://github.com/konnic/skeletons
+      ref: aa3237babe1e36457a6d09eb0d3889cdde97d89e
+      # Note: Custom fork for Flutter 3.22.0 compatibility (PR #12 pending)
+
+  # File and Storage
+  file_picker: ^9.0.2
+  flutter_cache_manager: ^3.4.1
+  flutter_file_dialog: ^3.0.2
   path_provider: ^2.1.5
+  shared_preferences: ^2.5.2
+
+  # Networking
+  dio: ^5.4.3+1
+  dio_cache_interceptor: ^3.5.0
+
+  # Device and System
+  connectivity_plus: ^6.1.3
+  device_apps:
+    git:
+      url: https://github.com/ponces/flutter_plugin_device_apps
+      ref: 0609662324b9e00931fccfa87e9e1b00b7610907
+      # Note: Custom fork for enhanced device app integration
```

Comment:
The note is also entirely wrong, we need ReVanced specific changes so the original maintainer forked it.